src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -61,4 +82,26 @@ public CreateCaseCallbackService(
             .filter(item -> !Strings.isNullOrEmpty(item.getTransformationUrl()))
             .getOrElse(Validation.invalid("Transformation URL is not configured"));
     }
+
+    private Validation<Seq<String>, Map<String, Object>> createNewCase(
+        ExceptionRecord exceptionRecord,
+        ServiceConfigItem configItem,
+        long caseId
+    ) {
+        try {
+            log.info("Start creating exception record for {} {}", configItem.getService(), caseId);
```

Comment:
Worth adding `service` and `case ID` words here and in the error log to make the message obvious.